Automattic/wp-calypso
# Migration: Lock target site while a migration is in progress

Note: This change is behind a feature flag and is part of an in-progress larger project. For context, see pbkcP4-8-p2.

#### Changes proposed in this Pull Request

* Updates the site state to persist migration status on update
* Updates selector to redirect to migration status page during an active migration

#### Testing instructions



* Run D35642-code on your sandbox.
* Start a migration from `/migrate`.
* Verify that while the migration is in progress, navigating to any other site path returns you to the migration progress screen
* Verify that you can again navigate once the migration finished
